requests-for-comments/2023-10-31-onboarding-emails.md

```diff
@@ -22,50 +22,52 @@ This campaign triggers when a user completes the `user signed up` event for the
 3. Check users' role in organization, using the `role_at_organization` property.
     1. If `founder` then wait 12 hours, and send Joe's Newsletter Invite email.
         1. Check if user clicked the subscription CTA.
-            1. If yes, add `newsletter_cta_clicked: true` to user.
-            2. If no, do nothing. 
+            1. If `Yes`, add `newsletter_cta_clicked: true` to user.
+            2. If `No`, do nothing. 
         2. Wait 12 hours
     2. If `engineering` then wait 12 hours, and send Andy's Newsletter Invite email. 
         1. Check if user clicked the subscription CTA.
-            1. If yes, add `newsletter_cta_clicked: true` to user.
-            2. If no, do nothing. 
+            1. If `Yes`, add `newsletter_cta_clicked: true` to user.
+            2. If `No`, do nothing. 
         2. Wait 12 hours
     3. If `product` then wait 12 hours. 
         1. Check if the user has performed `action created`
-            1. If yes, do nothing.
-            2. If no, send Advice for Product Teams email.
+            1. If `Yes`, send NEW email promoting the user interview survey template
+            2. If `No`, send Advice for Product Teams email.
         3. Wait 12 hours.
     4. If `marketing` then wait 12 hours. 
         1. Check if the user has performed `action created`
-            1. If yes, do nothing.
-            2. If no, send Advice for Marketing Teams email.
+            1. If `Yes`, send NEW email promoting the dashboard templates. 
+            2. If `No`, send Advice for Marketing Teams email.
         3. Wait 12 hours.
     5. If `sales` then wait 12 hours. 
         1. Check if the user has performed `action created`
-            1. If yes, do nothing.
-            2. If no, send Advice for Sales Teams email.
+            1. If `Yes`, send NEW email promoting the tutorials section.
+            2. If `No`, send Advice for Sales Teams email.
         3. Wait 12 hours.
     6. If not `founder` `engineering` `product` `marketing` or `sales` then wait 24 hours
 4. Wait until a weekday between 1PM and 5PM in the users' time zone (UTC fallback)
 5. Send AARRR intro article email
 6. Wait 1 week.
 7. Wait until a weekday between 1PM and 5PM in the users' time zone (UTC fallback)
 8. Check if the user is in the `Subscribers to Session Replays` segment
-    1. If `Yes`, do nothing. (See Session replay onboarding flow)
-    2. If `No`, assign to a random cohort branch.
-        1. 50% receive Session replay upsell email
-        2. 50% recieve Experiment: Personal Invite email
-9. Wait 1 week.
+    1. If `Yes`, check if the user is in the `Subscribers to Feature Flags` segment
+        1. If `Yes`, if the user is in the `Subscribers to Surveys` segment
+            1. If `Yes`, send [New email about using templates]
+            2. If `No`, send Survey Upsell email.
+        2. If `No`, send Feature Flag Upsell email
+    2. If `No`, send Replay Upsell email. 
```

Comment:
What if they are sub'd to Replays, but not product analytics? That seems like the next best sell in that situation.

Reply:
Yeah, we don't currently check analytics anywhere in this flow because we generally just assume users will already be aware and subscribing to it. But that's a dangerous assumption. 

Let me rewrite and rebuild with analytics factored in.